<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="logshow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>68</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>未开启服务</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>19</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>19</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startServiceBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开启服务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>68</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务状态：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopServiceBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止服务</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="onlineUserNum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
